src/utils/jsmnx.c

```diff
@@ -24,42 +24,46 @@
   if it is too small.  `num_tokens_ptr` should point to the number of
   allocated tokens.
 
-  Returns JSMN_ERROR_NOMEM on allocation error.
+  Returns number of tokens used by the parser or one of the following error
+  codes on error:
+    - JSMN_ERROR_NOMEM on allocation error.
+    - JSMN_INVAL on invalid character inside json string.
  */
 int jsmn_parse_alloc(jsmn_parser *parser, const char *js, const size_t len,
                      jsmntok_t **tokens_ptr, unsigned int *num_tokens_ptr)
 {
   int n, n_save;
-  unsigned int saved_pos;
   jsmntok_t *t=NULL;
   (void) n_save;  // avoid unused parameter error when assert is turned off
   assert(tokens_ptr);
   assert(num_tokens_ptr);
```

Comment:
Could you rename n to n_tokens, and t to tokens?

Reply:
done